Jenkinsfile

```diff
@@ -55,8 +54,8 @@ for (int i = 0; i < platforms.size(); ++i) {
 /* Execute our platforms in parallel */
 parallel(branches)
 
-
-stage('Verify demos')
+// TODO Restore once JENKINS-56632 is done
```

Comment:
I stated it in another PR but given it's currently commented, I would remove this. We have now the automated tests to do the verification job for us.

Reply:
I'm removing it in https://issues.jenkins-ci.org/browse/JENKINS-56632, which is already in progress, so I'll leave it here until then